safe-global/safe-wallet-web
# [TX flow] Adds nonce validation

## What it solves

Resolves #2110 

## How this PR fixes it

- Displays an error message in a tooltip around the nonce form
- Always displays the reset button but in a disabled state to prevent the input from jumping

## How to test it

1. Open the Safe
2. Create a new transaction
3. Choose a nonce lower than the current safe nonce
4. Observe an error inside a tooltip

## Screenshots
<img width="746" alt="Screenshot 2023-06-26 at 17 08 10" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/9221f90e-2aa6-4ce3-b87b-71ec0da077f2">

## Checklist
